command/push_test.go

```diff
@@ -123,15 +121,18 @@ func TestPush_goodBackendInit(t *testing.T) {
 		// Expected weird behavior, doesn't affect unpackaging
 		".terraform/",
 		".terraform/",
-		".terraform/plugins/",
-		fmt.Sprintf(".terraform/plugins/%s_%s/", runtime.GOOS, runtime.GOARCH),
-		fmt.Sprintf(".terraform/plugins/%s_%s/lock.json", runtime.GOOS, runtime.GOARCH),
+
+		// this comnfig contains no plugins
```

Comment:
ooh thanks, I did fix that, I but it leaked into my next branch. Will push rebase it into a new commit.